commons/src/main/res/values-az/strings.xml

```diff
@@ -48,7 +48,6 @@
     <string name="no_phone_call_permission">App does not have the permission to initiate phone calls, please grant it in the device settings</string>
     <string name="insert_text_here">Insert text here</string>
     <string name="call_person">%s şəxsinə zng et</string>
-    <string name="confirm_calling_person">Confirm calling %s</string>
```

Comment:
avoid removing existing strings please

Reply:
This is quite strange as I don't even recall deleting any of these string.